src/foam/core/AbstractDatePropertyInfo.java

```diff
@@ -0,0 +1,21 @@
+package foam.core;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZoneId;
+import java.util.Date;
+
+public abstract class AbstractDatePropertyInfo
+  extends AbstractPropertyInfo
+{
+
+  public int compareValues(java.util.Date o1, java.util.Date o2) {
```

Comment:
It seemed to be consistent amongst all of the other abstract types. The method allows you to compare dates in order to find the most recent date.